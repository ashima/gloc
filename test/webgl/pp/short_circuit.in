#if 3 == 3 || W
void w() {}
#endif

#if 4 || J
void j() {}
#endif

#if 0 && Q
void q() {}
#endif

#if 0 && Y
void y() {}
#endif

#if 1 && F
void f() {}
#endif

#if 0 || G
void g() {}
#endif
