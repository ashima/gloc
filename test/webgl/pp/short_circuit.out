File 0, line 17, col 9 - 10:
ambiguous preprocessor conditional branch: F

File 0, line 21, col 9 - 10:
ambiguous preprocessor conditional branch: G

Fatal: unrecoverable preprocessor divergence (4)
